<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,10,5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Libraries</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="libraryListWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="createLibraryBtn">
              <property name="text">
               <string>Create library</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="searchOnlineBtn">
              <property name="text">
               <string>Search online</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportLibraryBtn">
              <property name="text">
               <string>Export library</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="installLibraryBtn">
              <property name="text">
               <string>Install library</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="linkLibraryBtn">
              <property name="text">
               <string>Link library</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="unregisterLibraryBtn">
              <property name="text">
               <string>Unregister library</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="unregisterComponentBtn">
            <property name="text">
             <string>Unregister components</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="registerComponentBtn">
            <property name="text">
             <string>Register component</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="refreshLibraryBtn">
            <property name="text">
             <string>Refresh library</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="logTxt"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="contextTreeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="editPipDependenciesBtn">
            <property name="text">
             <string>Edit pip dependencies</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editMatDependenciesBtn">
            <property name="text">
             <string>Edit mat dependencies</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editSetupFileBtn">
            <property name="text">
             <string>Edit setup file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openContextBtn">
            <property name="text">
             <string>Open context</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
